src/components/forms/InstanceConfigurationRow.tsx

```diff
@@ -12,16 +12,20 @@ import { LxdProfile } from "types/profile";
 import { figureInheritedValue } from "util/instanceConfigInheritance";
 import { getConfigurationRow } from "components/ConfigurationRow";
 import { FormikProps } from "formik/dist/types";
+import { fetchConfigOptions } from "api/server";
+import { toConfigFields } from "util/config";
+import { ProjectFormValues } from "pages/projects/CreateProject";
+import { getProjectKey } from "util/projectConfigFields";
+import { getInstanceKey } from "util/instanceConfigFields";
 
 interface Props {
-  formik: SharedFormikTypes;
+  formik: SharedFormikTypes | FormikProps<ProjectFormValues>;
   name: string;
   label: string | ReactNode;
   children: ReactElement;
   defaultValue?: string | CpuLimit | MemoryLimit;
   disabled?: boolean;
   readOnlyRenderer?: (value: unknown) => string | ReactNode;
-  help?: string;
 }
 
 export const getInstanceConfigurationRow = ({
```

Comment:
I am not happy with the whole config row and table architecture, either. Was thinking about this a while ago and it should be possible to simplify the components. I'll add a ticket to make this nicer. Currently, I'd keep it as is though -- for renaming I have trouble finding a good name. I guess when refactoring, the right components with proper names would pop out of that process.

Reply:
As you prefer.

Reply:
Ah, wait - nevermind. I see that we have another `getConfigurationRow` with different semantics, and even another `ConfigurationTable`. It's more tangled than I thought. Feel free to keep it as it is - possibly with a short term plan of improving it so that we don't have `Instance` things used by project and so on.